var/spack/repos/builtin/packages/intltool/package.py

```diff
@@ -36,7 +36,7 @@ class Intltool(AutotoolsPackage):
     version('0.51.0', '12e517cac2b57a0121cda351570f1e63')
 
     # requires XML::Parser perl module
-    # depends_on('perl@5.8.1:', type='build')
+    depends_on('perl-xml-parser', type=('build', 'run'))
```

Comment:
Ok, I'll change, but is it a general  rule for all kind of dependencies?
In python extensions, python is not mentioned as dependency, is it because is the PythonPackage class that takes care of?
I' m trying to build turbovnc vncserver on spack X11
I' ve added dependencies up when ldd was not getting system libraries.
Currently it compiles but refuse to start as does not find some kbd files in share/X11/kbd
I think this is related to something not properly handled down in the X11 stack:
things get fixed by manually adding links from different package <prefix>/share/X11
could it be related to not explicitly mention all the possible X11 dependencies in turbovnc?
Thanks